manifests/init.pp

```diff
@@ -5,14 +5,17 @@
 # by an example created by Rackspace's cloudbuilders
 #
 class drbd(
-  $service_enable = true
-) {
+  $service_enable  = true,
+  $version         = $::drbd::params::version,
```

Comment:
I'm not using $version anywhere yet...
The first place I've noticed it'll be useful is [here](https://github.com/voxpupuli/puppet-drbd/blob/master/templates/global_common.conf.erb#L5)